packages/file-upload/src/utils.tsx

```diff
@@ -0,0 +1,7 @@
+import formatFileSize from 'filesize';
+
+export const getFilesTotal = (files: { size: number }[]) => {
+	const label = files.length > 1 ? 'files' : 'file';
+	const size = files.reduce((memo, { size }) => memo + size, 0);
```

Comment:
```suggestion
	const size = files.reduce((a, { size }) => a + size, 0);
```

I think `memo` is a bit confusing